frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -79,30 +85,33 @@ export const ObjectServiceDetailsCard: React.FC<DashboardItemProps> = ({
       </DashboardCardHeader>
       <DashboardCardBody>
         <DetailsBody>
-          <DetailItem
-            key="service_name"
-            title="Service Name"
-            value="OpenShift Container Storage"
-            isLoading={false}
-          />
+          <DetailItem key="service_name" title="Service Name" error={false} isLoading={false}>
+            OpenShift Container Storage
+          </DetailItem>
           <DetailItem
             key="system_name"
             title="System Name"
-            value={systemName}
             isLoading={!queryResult}
-          />
+            error={systemError}
+          >
+            <ExternalLink href={systemLink} text={systemName} />
```

Comment:
@rawagner If `DetailItem` has `error` prop set to a truthy value, does the child content (i.e. `ExternalLink` above) actually render?

If yes, the `systemLink` will be either invalid or undefined (see my suggestion above).

Reply:
no, child is not rendered then. `DetailItem` will render `unavailable` string

Reply:
Good, so we can simply use the `error={!systemLink}` as a guard for the system link :smiley:

Reply:
ack